packages/expect-type/src/__tests__/index.test.ts

```diff
@@ -183,6 +183,18 @@ test('More examples of ways to work with functions - parameters using `.paramete
   expectTypeOf(twoArgFunc).parameters.toEqualTypeOf<[number, string]>()
 })
 
+test('You can also check type guards & type assertions', () => {
+  const assertNumber = (v: any): asserts v is number => {
+    if (typeof v !== 'number') {
+      throw new TypeError('Nope !')
+    }
+  }
+  expectTypeOf(assertNumber).asserts.toBeNumber()
+
+  const isError = (v: any): v is string => typeof v === 'string'
```

Comment:
Shouldn't this be named `isString`?

Reply:
Dammit.

Reply:
Fixed in 30b3c4ee6149be26eb12d4646b971a6af2076082